lib/adminable.ex

```diff
@@ -27,11 +27,20 @@ defmodule Adminable do
     forward("/", Adminable.Plug, [
       otp_app: :my_app,
       repo: MyApp.Repo,
-      schemas: [MyApp.User]
+      schemas: [MyApp.User],
+      view_module: MyAppWeb.Adminable.AdminView
```

Comment:
Should we add this to the example in the `README.md` too for copy/paste purposes? Or maybe add some additional sentences to the "Exporting View and Templates" to remind them to set the `view_module`?

Reply:
You are correct. Updating Readme!